src/components/contract/ContractList.tsx

```
@@ -20,42 +20,49 @@ export const ContractList: React.FC = () => {
+              </Box>
+              <Box>
+                <Typography variant="subtitle1">Deployed Contracts</Typography>
+                <Stack>
+                  {isLoading ? (
+                    <Stack direction="row" justifyContent="center" paddingY={2}>
+                      <CircularProgress />
+                    </Stack>
+                  ) : (deployedContracts as Contract[])?.length ? (
+                    (deployedContracts as Contract[]).map((contract: Contract) => (
+                      <>
+                        <ContractItem address={contract.address} name={contract.name} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)